Targets/Apps/Windows_Notepad.tkape

```diff
@@ -0,0 +1,15 @@
+Description: Microsoft Windows 11 Notepad history
```

Comment:
https://github.com/EricZimmerman/KapeFiles/blob/master/Targets%2FWindows%2FNotepad.tkape

This Target appears to be duplicative. Please check if the existing one needs to be adjusted/updated!

Reply:
Sorry, you can dismiss it. I don't know why, after cancelling the previous pull, it was included in this last pull. :)